pkg/sdk/identifier_helpers.go

```diff
@@ -123,7 +123,7 @@ func NewAccountIdentifier(organizationName, accountName string) AccountIdentifie
 
 func NewAccountIdentifierFromAccountLocator(accountLocator string) AccountIdentifier {
 	return AccountIdentifier{
-		accountLocator: accountLocator,
+		accountLocator: strings.Trim(accountLocator, `"`),
```

Comment:
a bit afraid of it; let's leave it for the identifiers rework

Reply:
Removed trim